packages/kestrel_core/src/kestrel/ir/instructions.py

```diff
@@ -120,6 +120,7 @@ def resolve_references(self, f: Callable[[ReferenceValue], Any]):
 @dataclass(eq=False)
 class ProjectEntity(SolePredecessorTransformingInstruction):
     entity_type: str
+    native_type: Optional[str]
```

Comment:
We may remove `Optional` since we always has `native_type` given when created a `ProjectEntity`